index.php

```diff
@@ -0,0 +1,131 @@
+<?php
+require 'apicall.php';
+/**
+ * Class fetch to fetch the data from API.
+ * 
+ * Class for fetching title,service,image,alias and icon 
+ */
+class fetch
+{
+  public $title;
+  public $service;
+  public $image;
+  public $alias;
+  public $iconArr;
+  /**
+   * fetch constructor.
+   * 
+   * @param string $alias,$title,$service,$image,$iconArr
+   * 
+   * Fetched data will be passed as parameter.
+   */
+  function __construct($title, $service, $image, $alias, $iconArr)
+  {
+    $this->title = $title;
+    $this->service = $service;
+    $this->image = $image;
+    $this->alias = $alias;
+    $this->iconArr = $iconArr;
+  }
+  /**
+   * iconArrLen function to count the number of icons which will be stored in array format.
+   *
+   * @return integer
+   */
+  function iconArrLen()
+  {
```

Comment:
Function declaration should follow this formatting. Make these changes everywhere.
```suggestion
  function iconArrLen() {
```